### ISSUE TITLE
Is there other way to sponsor?

### ISSUE THREAD

**Issue by nowislewis** [dalanicolai/pdf-continuous-scroll-mode.el#17]
Thank you very much for your work, I have encountered some problems. Is there any other way? There is no doubt that you made a great work, thank you dalanicolai. All the things really help me a lot.
<img width="1412" alt="截屏2022-02-16 上午12 18 31" src="https://user-images.githubusercontent.com/41566630/154104254-ddd5ea41-b5fe-4b92-8ea3-2d930546a1c9.png">

**Comment by dalanicolai** [comment]
Thank you for reaching out, it is already nice (stimulating) to hear that you find it useful. Besides the other option of donating via PayPal, I did not yet set up other alternatives ways for donating. However, I guess that I will create a Github Sponsors profile (very) soon. I will add a notification here when it is ready.

I am happy to tell you that, after many frustrating hours/days of rewriting but especially debugging the package/Emacs, I did make some progress (finding out how the display mechanism works took 98% of the time, as it seemed complex while it is not documented, and some Emacs bug kept it from working correctly). I guess/hope that soon a basic version will be 'stable' enough for merging with pdf-tools (although I am not sure how much work it will be to make it compatible with all the `pdf-tools-enabled-modes`).
